packages/block-editor/src/components/block-mover/index.native.js

```diff
@@ -10,7 +10,31 @@ import { ToolbarButton } from '@wordpress/components';
 import { __, sprintf } from '@wordpress/i18n';
 import { withSelect, withDispatch } from '@wordpress/data';
 import { withInstanceId, compose } from '@wordpress/compose';
-import { arrowUp, arrowDown } from '@wordpress/icons';
+import { arrowUp, arrowDown, arrowLeft, arrowRight } from '@wordpress/icons';
+
+const horizontalMover = {
+	firstButtonIcon: arrowLeft,
+	secondButtonIcon: arrowRight,
+	firstButtonHint: __( 'Double tap to move the block to the left' ),
+	secondButtonHint: __( 'Double tap to move the block to the right' ),
+	firstBlockTitle: __( 'Move block left' ),
+	lastBlockTitle: __( 'Move block right' ),
+	firstButtonDirection: 'left',
+	secondButtonDirection: 'right',
+	location: 'position',
```

Comment:
These are left not translated, translations won't work when we inject these as variables. We need to explicitly wrap them with __( ), because they are parsed in compile time

Reply:
done.

We have spoke recently about the translation. What's the proper way to test if this translation works as expected ? Change the device language and see the output ?

Reply:
> Change the device language and see the output ?

This will only work for already existing translations. Unfortunately there's no easy way to test the new ones. We should do our best to send easy to understand sentences by providing enough context in the code comments: `/* translators: ...*/`

Reply:
I have made changes and post my answer [here](https://github.com/WordPress/gutenberg/pull/19013#discussion_r403943436)